concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java

```diff
@@ -132,6 +134,21 @@ public Iterator<AccountAddress> getAccountList(final BlockHashInput input) {
         return to(grpcOutput, ClientV2MapperExtensions::to);
     }
 
+    /**
+     * Retrieves the next {@link AccountNonce} for an account.
+     * This is the {@link AccountNonce} to use for future transactions
+     * E.g. when using {@link Client#sendTransaction(Transaction)}
+     *
+     * @param address The {@link AccountAddress}
+     * @return The next {@link AccountNonce}
+     */
+    public AccountNonce getNextAccountSequenceNumber(AccountAddress address) {
```

Comment:
Please state what the outcome will be of querying an "invalid" account. I.e. an account that does not exist on chain from the point of view of the node.

Reply:
nonce=1, is returned by the function

Reply:
This is the expected behavior. But I think it would be good to document. 
I.e. a small note that "when this function is queried with a non existent account it will return (1,True).